queries.sql

```diff
@@ -42,3 +42,18 @@ SELECT AVG(weight_kg) FROM animals;
 SELECT neutered, COUNT(escape_attempts) FROM animals GROUP BY neutered;
 SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
 SELECT species, AVG(escape_attempts) FROM animals  WHERE date_of_birth > '1990-01-01' AND date_of_birth <'2000-12-31'  GROUP BY species;
+
+SELECT animals.name FROM animals JOIN owners ON owner_id = owners.id WHERE owners.id = 4;
+SELECT animals.name FROM animals Join species ON species_id = species.id WHERE species.id = 1;
+SELECT owners.full_name, animals.name FROM owners RIGHT JOIN animals ON owner_id = owners.id;
+SELECT species.name, COUNT(*) FROM animals JOIN species ON species_id = species.id GROUP BY species.name;
+SELECT animals.name FROM animals
+JOIN species ON species_id = species.id
+JOIN owners ON owner_id = owners.id 
+WHERE owners.full_name = 'Jennifer Orwell';
```

Comment:
- [ ] Kindly list only animals whose species are Digimon. This query displays all animals belonging to Jennifer Orwell. Pikachu is not a Digimon. Kindly fix this, please.
consider using `WHERE owners.fullname = 'Jennifer Orwell' AND species.name = 'Digimon'`